CI/physmon/physmon.py

```diff
@@ -333,18 +339,17 @@ def run_vertexing(fitter, mu, events):
             s,
             trackingGeometry,
             field,
-            CKFPerformanceConfig(ptMin=400.0 * u.MeV, nMeasurementsMin=6),
-            TrackSelectorRanges(
+            TrackSelectorConfig(
                 pt=(500 * u.MeV, None),
                 loc0=(-4.0 * u.mm, 4.0 * u.mm),
                 absEta=(None, 3.0),
+                nMeasurementsMin=6,
```

Comment:
Is there a strong reason to duplicate the reco job setup in the physmon or can we find a better way to avoid duplicating configuration code?

Reply:
I think physmon and full chain is meant for different things. so they will diverge in the future but that is okay. I just align them right now so we can offer pulls of 1 for both